adiachenko/src/components/Film/index.tsx

```diff
@@ -19,26 +26,28 @@ const Film = ({film}: FilmsProps) => {
   const [isDeleteOpen, setDeleteOpen] = useState(false);
   const release = film.release_date.split('-')[0];
 
+  const contextValue = useAppContext();
+
   const hideActions = () => setActionsVisible(false);
-  const showActions = () => setActionsVisible(true);
+
+  const showActions = useCallback(() => setActionsVisible(true), []);
```

Comment:
Again, it's not something that can help you to benefit from `useCallback()`

Reply:
In this case, we have two pure components. And we send this function us a property.
If we will remove it in this case we will see the next log from "why should we render".
And we will have unwanted rerenders:
![image](https://user-images.githubusercontent.com/44708931/111916938-00498100-8a86-11eb-8af0-213f1b15ebb5.png)